packages/common/src/restHelpers.ts

```diff
@@ -993,7 +992,7 @@ export function getLayerUpdates(
   // forceRelationshipCheck should be true when layers are added in chunks.
```

Comment:
This comment line appears to be obsolete; also `forceRelationshipCheck` input argument documentation line.